db/seeds.rb

```diff
@@ -5,3 +5,59 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+user_data = [
+  { name: 'admin', email: 'admin@example.com', admin: true, password: 'password' },
+  { name: 'user', email: 'user@example.com', admin: false, password: 'password' },
+]
+
+user_data.each do |user|
+  puts "Seeding user: #{user[:name]} and #{user[:email]}"
+  library_user = User.create_with(password: user[:password]).
+    find_or_create_by(name: user[:name], email: user[:email], admin: user[:admin])
```

Comment:
Just to answer the question posted on slack... you had to move `password` into `create_with` here because `find_or_create_by` takes it's arguments and does a `find_by` first. You can't use `password` in a query because it's not an actual column on the table. When you set a password, clearance sets the `encrypted_password` column.